plugins/modules/junos_security_policies_global.py

```diff
@@ -0,0 +1,979 @@
+#     },
+#     "policy_stats": {
+#         "enable": true,
+#         "system_wide": true
+#     },
+#     "pre_id_default_policy_action": {
+#         "log": {
+#             "session_init": true
+#         },
+#         "session_timeout": {
+#             "icmp": 10,
+#             "others": 10
+#         }
+#     },
+#     "traceoptions": {
+#         "file": {
+#             "files": 3,
+#             "match": "/[A-Z]*/gm",
+#             "no_world_readable": true,
+#             "size": "10k"
+#         },
+#         "flag": "all",
+#         "no_remote_trace": true
+#     }
+# }
+#
+#
+"""
+RETURN = """
+before:
+  description: The configuration prior to the module execution.
+  returned: when state is I(merged), I(replaced), I(overridden), I(deleted) or I(purged)
+  type: dict
+  sample: >
+    This output will always be in the same format as the
+    module argspec.
+after:
+  description: The resulting configuration after module execution.
+  returned: when changed
+  type: dict
+  sample: >
+    This output will always be in the same format as the
+    module argspec.
+commands:
+  description: The set of commands pushed to the remote device.
+  returned: when state is I(merged), I(replaced), I(overridden), I(deleted) or I(purged)
+  type: list
+  sample:
+    - "<rpc-reply>
+      <configuration>
+        <security>
+          <policies>
+            <default-policy>
+              <permit-all />
+            </default-policy>
+          </policies>
+        </security>
+      </configuration>
+    </rpc-reply>"
+rendered:
+  description: The provided configuration in the task rendered in device-native format (offline).
+  returned: when state is I(rendered)
+  type: dict
+  sample:
+    - "<rpc-reply>
+      <configuration>
+        <security>
+          <policies>
+            <default-policy>
+              <permit-all />
+            </default-policy>
+          </policies>
+        </security>
+      </configuration>
+    </rpc-reply>"
+gathered:
+  description: Facts about the network resource gathered from the remote device as structured data.
+  returned: when state is I(gathered)
+  type: dict
+  sample: >
+    This output will always be in the same format as the
+    module argspec.
+parsed:
+  description: The device native config provided in I(running_config) option parsed into structured data as per module argspec.
+  returned: when state is I(parsed)
+  type: dict
+  sample: >
+    This output will always be in the same format as the
+    module argspec.
+"""
+
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.junipernetworks.junos.plugins.module_utils.network.junos.argspec.security_policies_global.security_policies_global import (
+    Security_policies_globalArgs,
+)
+from ansible_collections.junipernetworks.junos.plugins.module_utils.network.junos.config.security_policies_global.security_policies_global import (
+    Security_policies_global,
+)
+
+
+def main():
```

Comment:
Please add functionality to check for empty config for opearation it is required.,
also test needs to be added for that